_posts/2022-03-28-utah-lidar-coverage-application.md

```diff
@@ -0,0 +1,18 @@
+---
+title: 'Introduction: Utah Lidar Coverage Application'
+author:
+  display_name: Rick Kelson
+  email: rkelson@utah.gov
+date: 2022-03-28 09:00:00
+categories:
+  - Featured
+  - SGID Blog
+tags:
+  - lidar
+  - dem
+  - elevation
+---
+
+Since 2006 UGRC has partnered with various agencies to acquire public domain high resolution [LiDAR](https://oceanservice.noaa.gov/facts/lidar.html) (Light Detection and Ranging) elevation data for areas in Utah. In 2013 UGRC adopted the U.S. Geological Survey’s [Lidar Base Specifications](https://www.usgs.gov/ngp-standards-and-specifications/lidar-base-specification-online) (LBS) for lidar acquisitions and project deliverables. Adhering to the LBS allowed for the project deliverables in Utah to be incorporated into [The National Map](https://www.usgs.gov/programs/national-geospatial-program/national-map). In more recent years, UGRC added the [USGS 3D Elevation Program](https://www.usgs.gov/3d-elevation-program) (3DEP) to the list of partnering agencies. This allowed UGRC and local partners to leverage their dollars by submitting grant proposals and applying for additional 3DEP funding to vastly increase the number of square miles that could be purchased and acquired for each project cycle.
+
+To accompany our [web pages of lidar projects](https://gis.utah.gov/data/elevation-and-terrain/#LiDAR) and downloads from the [UGRC Raster Data Discovery](https://raster.utah.gov/) application, the [Utah Lidar Coverage](https://utah.maps.arcgis.com/apps/instant/minimalist/index.html?appid=35d08d7d23ca4293b657d570f1005822) application has been created. It depicts the lidar projects that have been proposed, are in progress, or have been completed in Utah. This will allow users to see the distribution of lidar projects over the years and their [USGS Quality Level](https://www.usgs.gov/3d-elevation-program/topographic-data-quality-levels-qls) in addition to other information and resources related to the projects
```

Comment:
**issue**: fix internal link

```suggestion
To accompany our [web pages of lidar projects]({% link data/elevation-and-terrain/index.html %})#LiDAR) and downloads from the [UGRC Raster Data Discovery](https://raster.utah.gov/) application, the [Utah Lidar Coverage](https://utah.maps.arcgis.com/apps/instant/minimalist/index.html?appid=35d08d7d23ca4293b657d570f1005822) application has been created. It depicts the lidar projects that have been proposed, are in progress, or have been completed in Utah. This will allow users to see the distribution of lidar projects over the years and their [USGS Quality Level](https://www.usgs.gov/3d-elevation-program/topographic-data-quality-levels-qls) in addition to other information and resources related to the projects
```